### PR TITLE
Added annotations links to PD and MMA

### PR DESCRIPTION
**Pull request by wwysoc2**

### CODE REVIEW

**Review by wteouchicago — APPROVED** [review]
Looks good to me.

**Review by wteouchicago — APPROVED** [review]
changes to MT look good too.

**Review by philtom-ctds — APPROVED** [review]

File: gdcdictionary/schemas/annotation.yaml [hunk 1]
```diff
@@ -166,6 +166,12 @@ links:
         target_type: gene_expression
         multiplicity: many_to_many
         required: false
+      - name: masked_methylation_arrays
```
Comment (philtom-ctds) [inline gdcdictionary/schemas/annotation.yaml:169]: also need to add these new links to `properties`

Comment (wwysoc2) [reply gdcdictionary/schemas/annotation.yaml:169]: done!

**Pull request merged**